src/include/server/mir/server.h

```diff
@@ -491,6 +491,7 @@ class Server
             wl_display*,
             std::function<void(std::function<void()>&& work)> const&)> builder);
 
+    [[deprecated("Prefer using set_wayland_extension_policy() to set a policy on a per-interface bassis.")]]
     void set_wayland_extension_filter(
         std::function<bool(std::shared_ptr<scene::Session> const&, char const*)> const& extension_filter);
```

Comment:
Any reason for keeping the deprecated functions? We break the mirserver ABI almost every time anyway